.eslintignore

```diff
@@ -1,2 +1,3 @@
 node_modules/*
 coverage/*
+test/*
```

Comment:
Just read the travis log and as you do not use babel or something like that, the node 4 & 5 build failed.

I will make the old class syntax, it will be easier for just a simple test ;)